Application/data_payload.hpp

```diff
@@ -62,6 +62,10 @@ struct DataPayload {
     	CsvString(buffer, length);
 	}
 
+    void RawRow(float* buffer){
+    	return RawArray(buffer);
```

Comment:
The `return` statement is not doing anything since `RawArray()` is a void function

Reply:
Nice catch, not sure why its there, fixed